relay-event-schema/src/protocol/breadcrumb.rs

```diff
@@ -116,6 +116,14 @@ pub struct Breadcrumb {
     #[metastructure(skip_serialization = "null")]
     pub event_id: Annotated<EventId>,
 
+    /// The origin of the breadcrumb. _Optional_, defaults to `null`.
```

Comment:
```suggestion
    /// The origin of the breadcrumb.
```

Usually don't mention that, it's implied. Also it will not default to null, Relay strips it.